### ISSUE TITLE
WARNING: No pools available: check your configuration

### ISSUE THREAD

**Issue by tcj2001** [RainbowMiner/RainbowMiner#2465]
Installed RainbowMiner on two different Ubuntu 22.04 machine.
Both of them are showing WARNING: No pools available: check your configuration for past couple of hours, no progress at all
[debug_2023-09-11.zip](https://github.com/RainbowMiner/RainbowMiner/files/12580588/debug_2023-09-11.zip)
[debug_2023-09-11 (1).zip](https://github.com/RainbowMiner/RainbowMiner/files/12580592/debug_2023-09-11.1.zip)

**Comment by tcj2001** [comment]
The debuf file show, its giving error on every API calls, I am able to ping the domain of the API's from the machine

**Comment by tcj2001** [comment]
Tried with use cUrl, still everything coming as
[2023-09-12 00:01:05] cURL 0 / HTTP 404
[2023-09-12 00:01:05] cURL 0 / HTTP 404
[2023-09-12 00:01:05] cURL 0 / HTTP 404
[2023-09-12 00:01:05] cURL 0 / HTTP 404
[2023-09-12 00:01:05] cURL 0 / HTTP 404
[2023-09-12 00:01:05] cURL 0 / HTTP 404
[2023-09-12 00:01:05] cURL 0 / HTTP 404
[2023-09-12 00:01:05] cURL 0 / HTTP 404
[2023-09-12 00:01:05] cURL 0 / HTTP 404
[2023-09-12 00:01:05] cURL 0 / HTTP 404
[2023-09-12 00:01:05] cURL 0 / HTTP 404
[2023-09-12 00:01:05] cURL 0 / HTTP 404

[2023-09-12 00:01:04] Error job 1292AA063F94DD68BB9C90469A615883 with https://zergpool.com/api/status using REST: Problem fetching https://zergpool.com/api/status using REST: cURL 0 / HTTP 404

**Comment by RainbowMiner** [comment]
Are you able to connect to the ZergPool URLs via your web browser?
It seems, like something is blocking access to the URLs. 404 means "not found, but I tried"

**Comment by RainbowMiner** [comment]
Also, is there any firewall installed, that might block the Rainbowminer calls?

**Comment by tcj2001** [comment]
i am able to ping the zergpool domain from the machine, also wrote a ps1 script to connect to coinbase api and get data back. Today I will reinstall ubuntu lts 22.04 and then install rainbowminer again and will let u know

**Comment by tcj2001** [comment]
I reinstalled Ubuntu and then RainbowMiner on both machine again, but still I am getting 
[2023-09-12 13:23:48] Error job 34D1FC8F1EB5271842739D69997C6B54 with https://api.github.com/repos/rainbowminer/RainbowMiner/releases/latest using REST: Problem fetching https://api.github.com/repos/rainbowminer/RainbowMiner/releases/latest using REST: 404 Not Found (Client Error)
[2023-09-12 13:23:49] Error job unprofitable3 with https://rbminer.net/api/data/unprofitable3.json using REST: Problem fetching https://rbminer.net/api/data/unprofitable3.json using REST: 404 Not Found (Client Error)
[2023-09-12 13:23:49] Error job unprofitablecpu with https://rbminer.net/api/data/unprofitable-cpu.json using REST: Problem fetching https://rbminer.net/api/data/unprofitable-cpu.json using REST: 404 Not Found (Client Error)
[2023-09-12 13:23:49] Error job coinbase with https://api.coinbase.com/v2/exchange-rates?currency=BTC using REST: Problem fetching https://api.coinbase.com/v2/exchange-rates?currency=BTC using REST: 404 Not Found (Client Error)

**Comment by tcj2001** [comment]
Finally after lot of debuging found there was issue in the way I setup the servers. I have total 3 servers, first one I installed as standalone and it worked fine, the other two i setup as client and made the first server as "server", which i beleive is not correct, to use a server it should be setup as "server" not standalone. 
so in my case all the jobs were send to sever which was StandAlone and it was giving 404 error on all api calls.

So now I set up all the servers as stansdalone and all are working fine.
